docker/debug/go.mod

```diff
@@ -0,0 +1,29 @@
+module debug-delve
+
+go 1.23.0
+
+require (
+	github.com/cilium/ebpf v0.11.0 // indirect
+	github.com/cosiner/argv v0.1.0 // indirect
+	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
+	github.com/derekparker/trie v0.0.0-20230829180723-39f4de51ef7d // indirect
+	github.com/go-delve/delve v1.22.1 // indirect
```

Comment:
Why is this indirect dep? In internal/tools/go.mod we have a section for the actual dependencies, can we use the same structure?